libraries/Preferences/examples/Prefs2Struct/Prefs2Struct.ino

```diff
@@ -0,0 +1,43 @@
+/*
+This example shows how to use Preferences (nvs) to store a
+structure.  Note that the maximum size of a putBytes is 496K
```

Comment:
@lbernstone Are you sure the 496K limit is accurate? I thought NVS's limit was much smaller. I though all NVS data blocks were limited to fit inside one 4096 BYTE FLASH page? Base on the standard NVS configs:
```c++
// From IDF nvs_page.hpp
 const uint32_t PSB_INIT = 0x1;
 const uint32_t PSB_FULL = 0x2;
 const uint32_t PSB_FREEING = 0x4;
 const uint32_t PSB_CORRUPT = 0x8;

 const uint32_t ESB_WRITTEN = 0x1;
 const uint32_t ESB_ERASED = 0x2;

 const uint32_t SEC_SIZE = SPI_FLASH_SEC_SIZE;

 const size_t ENTRY_SIZE  = 32;
 const size_t ENTRY_COUNT = 126;
 const uint32_t INVALID_ENTRY = 0xffffffff;
    
 const size_t BLOB_MAX_SIZE = ENTRY_SIZE * (ENTRY_COUNT / 2 - 1);

 const uint8_t NS_INDEX = 0;
 const uint8_t NS_ANY = 255;
```
Using this calculation, max Blob Size is 1984 Bytes?

Chuck.

Reply:
pinging @lbernstone :D

Reply:
I am going by what the docs say: https://docs.espressif.com/projects/esp-idf/en/latest/api-reference/storage/nvs_flash.html#keys-and-values.  I didn't test, but I suppose someone (me) should.

Reply:
I don't have psram on my test rig here, but I can put as much into putBytes as a I can get into a char array (113792 bytes tested).  Satisfies me enough to not put the energy into writing something to try to stream more :smile: